src/ir/dot/dot_generator.cc

```diff
@@ -57,14 +57,14 @@ class DotGeneratorHelper : public IRTraversingVisitor<DotGeneratorHelper> {
   std::string GetNodeName(const Operation& operation) {
     const Block* block = operation.parent();
     auto id = ssa_names_.GetOrCreateID(operation);
-    return absl::StrFormat("%s_%d",
+    return absl::StrFormat("%s_%s",
```

Comment:
Enclose the node name in quotes (`"`):
```suggestion
    return absl::StrFormat("\"%s_%s\"",
```

Reply:
If you do this, you might want to make it a raw string to avoid the escaping if that makes things more readable.

Reply:
Done!